app/backend/src/clj/teet/migration/file_part_statuses.clj

```diff
@@ -0,0 +1,93 @@
+(ns teet.migration.file-part-statuses
+  (:require [datomic.client.api :as d]
+            [clojure.string :as string]
+            [teet.util.collection :as cu]
+            [teet.util.datomic :as du]))
+
+(def user-info [:db/id :user/person-id :user/given-name :user/family-name])
+
+(defn- get-task-meta
```

Comment:
this fn looks like a verbatim copy of `entity-tx-log` in admin queries, why? Use that directly or query only what you need...

Reply:
My guess was that for the migration it would be safer to copy the function there should admin queries be changed, but chance of that is slim so I will call directly.